index.html

```diff
@@ -1002,6 +1002,39 @@ <h2 class='informative'>Implementation Patterns</h2>
       </section>
     </section>
 
+    <section id="privacy">
+      <h2>Privacy Considerations</h2>
+      <p>
+        Fedora resources may contain personally identifiable information such as the identify of
+        the agents who create and modify resources. As such, this information could be analyzed
+        to generate profiles of the behavior of individuals or groups.
+      </p>
+      <p>
+        As such, Fedora implementations that produce or consume personally identifiable
+        information MUST clearly document the type and extent of personal information that
+        is collected and published along with the reasons for doing so.
```

Comment:
@ajs6f an implementation would do this via documentation. For example, an implementation's documentation might read:

> User data is stored in a resource's audit record. Administrators should take care when exposing such information to the public web.

That would suffice, in my opinion.

That said, the text here is intended simply as a starting point, and if you have different ideas about a "Privacy" section (including dropping the section entirely), I would welcome comment.